jcommune-view/jcommune-web-view/src/test/java/org/jtalks/jcommune/test/UserControllerTest.groovy

```diff
@@ -44,153 +37,118 @@ import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.
 @TransactionConfiguration(transactionManager = "transactionManager", defaultRollback = true)
 @Transactional
 class UserControllerTest extends Specification {
-    @Autowired ComponentService componentService
     @Autowired Users users
     @Autowired GroupsService groupsService
 
-    @Autowired MockMvc mockMvc
-
-    private Component forum
-    private User adminUser;
-
     def setup() {
         groupsService.create()
-        forum = componentService.createForumComponent()
-
-        adminUser = new User()
-        users.created(adminUser).withPermissionOn(forum, GeneralPermission.ADMIN)
     }
 
-    def 'test show user groups'() {
+    def 'test must return all the groups of a user'() {
```

Comment:
Do not include word 'test' - don't forget that this is a description of the behaviour, not a test :)